src/WcfCoreMtomEncoder/MtomPart.cs

```diff
@@ -18,7 +18,20 @@ public MtomPart(HttpContent part)
             _part = part;
         }
 
-        public MediaTypeHeaderValue ContentType => _part.Headers.ContentType;
+        public MediaTypeHeaderValue ContentType
+        {
+            get
+            {
+                string contentTypeHeaderValue = _part.Headers.GetValues("Content-Type").FirstOrDefault();
+
+                MediaTypeHeaderValue parsedValue;
+                if (!String.IsNullOrEmpty(contentTypeHeaderValue) && MediaTypeHeaderValue.TryParse(contentTypeHeaderValue.TrimEnd(';'), out parsedValue))
```

Comment:
Could you remove the `parsedValue` declaration, and change the out value to `out var parsedValue` to tidy things up a bit

Reply:
Sure!